<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BluetoothDialog</class>
 <widget class="QDialog" name="BluetoothDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(175, 215, 255);</string>
  </property>
  <widget class="QListWidget" name="listWidget_DiscoveredDevices">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>760</width>
     <height>320</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bluetooth devices</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>430</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Clean">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>430</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Search">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_BluetoothDiscoverable">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bluetooth discoverable</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Double click to pair device</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
